test/missing_dbf_columns_test.rb

```diff
@@ -17,7 +17,7 @@ def test_missing_columns
   def _open_shapefile(name_, &block_)
     RGeo::Shapefile::Reader.open(
       File.expand_path("missing_dbf_columns/#{name_}",
-      File.dirname(__FILE__)),
+                       File.dirname(__FILE__)),
```

Comment:
Interesting that rubocop doesn't "fix" this to use `__dir__` 🤔

Reply:
It use to do it in my company, I think it is a matter of configuration :)

Reply:
I think something is weird with my dev setup - this cop doesn't seem to be working 🤔
Any ideas?

Reply:
Currently looking at that.

First thing, I think we should add rubocop to dev dependencies!

Then:

```
Style/GlobalVars:
  Exclude:
    - 'ext/geos_c_impl/extconf.rb'
 ```
 
 makes no sense here :)
 
 And finally it seems like the rule takes into account a slightly different case than ours. I'd be in favor of searching and replacing, and that's it :)